ZacGormanrewrite.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
+<html>
+	<body>
+		<h1>
+			"I have a webcomic called Magical Game Time which i update super infrequently."
+			<section>
+				<p>
+				"I sometimes do storyboards and character design for Nickelodeon and Cartoon Network but mostly i wrte and draw comics like this book here"
+				</p>
+			</section>
+		</h1>
+			<img> href="http://36.media.tumblr.com/bb8a1bd2a9c9df3e7d5e347e110ff0d8/tumblr_nfyyhgDiLb1r3abdgo1_500.png"</img>
+		<section>	
+			<h2>
+				<li>Ask</li>
+				<img>http://36.media.tumblr.com/bb8a1bd2a9c9df3e7d5e347e110ff0d8/tumblr_nfyyhgDiLb1r3abdgo1_500.png</img>
+			</h2>
+		</section>
+		// Left out tumblr code
+
+	</body>
+</html>
```

Comment:
Comparing your work and the original site. I think you could have build this out a little more. If you need to use the Chrome inspector or view the source of the original page to get an idea of all the tags used in the building of that site.